test/topology_random_failures/cluster_events.py

```diff
@@ -420,7 +420,9 @@ async def remove_data_dir_of_dead_node(manager: ManagerClient,
                                        random_tables: RandomTables,
                                        error_injection: str) -> AsyncIterator[None]:
```

Comment:
Maybe rename the event as well, since now we're clearing not just the data dir but the whole workdir?
`remove_workdir_of_dead_node`

Or `remove_data_of_dead_node` (data as in all data, not just the "data dir")